lab_2_retrieval_w_bm25/tests/assets/saved_metrics_example.json

```diff
@@ -36,4 +36,4 @@
         "output": 0.0,
         "across": 0.3664
     }
-]
\ No newline at end of file
+]
```

Comment:
revert

Reply:
the mentor says everything is alright now, there is nothing to revert. So, I suppose, done